apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -56,7 +57,6 @@ export default function ListBoxInline({ app, fieldIdentifier, stateName = '$', o
     postProcessPages = undefined,
     calculatePagesHeight,
     showGray = true,
-    sortByState = 1,
```

Comment:
Still need to support options.sortByState so that we don't break the API.

Reply:
I added options.sortByState to the `DoNotUseOptions` not too long ago (may 30, 2022). Do you still think it's necessary to keep it?

Reply:
no, if it isn't official then it is fine to remove it.